src/main/java/sirius/db/es/LowLevelClient.java

```diff
@@ -294,7 +294,7 @@ public JSONObject count(List<String> indices, String type, String routing, JSONO
     @Explain("Due to method overloading the compiler cannot deduce which method to pick")
     public JSONObject bulk(List<JSONObject> bulkData) {
         return performPost().rawData(bulkData.stream().map(obj -> obj.toJSONString()).collect(Collectors.joining("\n"))
-                                     + "\n", "application/x-ndjson").execute("_bulk").response();
+                                     + "\n").execute("_bulk").response();
```

Comment:
When sending this content-type, the server responds with something like "content-type" is not set.
adapted it as seen in this PR https://github.com/dadoonet/fscrawler/pull/384/files